consumer-features/example-provider-hello-world/v3/example-provider-hello-world-js/src/provider.pact.spec.js

```diff
@@ -0,0 +1,30 @@
+const { Verifier } = require("@pact-foundation/pact");
+const { server } = require("./provider");
+
+describe("Pact Verification", () => {
```

Comment:
Think this one needs a
TODO: update bits to make it example specific

and also I'm not quite sure what to do with the provider part, how best to structure :thinking: consumer is easy enough with reference pacts to check against